metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -1084,7 +1084,7 @@ public RestoreIndicesResult restoreIndices(@Nonnull RestoreIndicesArgs args, @No
     logger.accept(String.format(
         "Reading rows %s through %s from the aspects table completed.", args.start, args.start + args.batchSize));
 
-    for (EbeanAspectV2 aspect : rows.getList()) {
+    for (EbeanAspectV2 aspect : rows != null ? rows.getList() : List.<EbeanAspectV2>of()) {
```

Comment:
Rows is null when cassandra

Reply:
Really? That's wild

Reply:
Cassandra will return null since it doesn't implement the paged method. Not new, but there seems to be a new test.